### DISCUSSION TITLE
[Released] Custom URL for preview deployments

### DISCUSSION THREAD

**Discussion by danaugrs** [vercel/community#85] [Idea]
[Update] https://vercel.com/docs/concepts/deployments/automatic-urls#preview-deployment-suffix

---

Is it possible to customize the URL for preview deployments?

E.g. instead of `<random_id>.vercel.app` have them go to `<random_id>.mydomain.com` automatically ?
Or at `featurename.mydomain.com` or even `mydomain.com`?

**Comment by jacobhq** [comment]
If you want to do it automatically, it's only available for teams on the pro plan. See here: https://vercel.com/docs/concepts/deployments/automatic-urls#preview-deployment-suffix. I haven't tried the alias, command, and can't find it in the docs, yet it is referenced on the dashboard.

**Edit** - fixed link

**Reply by tanguyantoine** [reply]
It looks like it is available only at the team level. 
How would you do that per project?

**Reply by tanguyantoine** [reply]
btw: link changed https://vercel.com/docs/concepts/deployments/automatic-urls#preview-deployment-suffix

**Comment by leerob** [comment]
This has been released: https://vercel.com/docs/concepts/deployments/automatic-urls#preview-deployment-suffix

**Reply by rdsedmundo** [reply]
Paying $100/mo for this feature on top of the Pro plan seats is nothing but ridiculous, I have implemented that on Lambda@Edge in the past with literally 10 lines of code. This left me with a bad impression from Vercel as it seems to be trying to pull a fast on others to gain easy money.
